vm/src/object/core.rs

```diff
@@ -114,9 +114,23 @@ struct PyInner<T> {
     dict: Option<InstanceDict>,
     weak_list: WeakRefList,
 
+    size: PyRwLock<usize>, // PyVarObject.ob_size
+
     payload: T,
 }
 
+impl<T> PyInner<T> {
+    #[inline(always)]
+    pub fn ob_size(&self) -> usize {
```

Comment:
In RustPython, we don't use the prefix `ob_`. 

```suggestion
    pub fn size(&self) -> usize {
```